samples/highcharts/blog/line-chart/demo.details

```diff
@@ -0,0 +1,6 @@
+---
+ name: Line chart to replace a chart race
```

Comment:
What does "Line chart to replace a chart race" mean? This is a static line chart, not a race?